addons/crm/tests/test_crm_lead_duplicated.py

```diff
@@ -0,0 +1,182 @@
+from odoo.tests.common import tagged
+from odoo.addons.crm.tests.common import TestCrmCommon
+
+
+@tagged('lead_duplicates')
+class TestLeadConvert(TestCrmCommon):
+
+    def check_duplicates(self, leads, k, keys):
```

Comment:
This is quite hard to understand ... what are you trying to assert here ? What are the expected values, what are you asserting ? This require some rewriting so that people can easily understand your asserts, improve them, understand what happened if there is an issue, ...